assets/docs/snippets/get-started.md

```diff
@@ -0,0 +1,30 @@
+2. Deploy the Kubernetes Gateway API CRDs.
```

Comment:
Created this conref bc it's required in the lambda irsa guide. Looks great in the quickstart, picks up at step 2. But on the lambda page I need it to start at step 1 and it does not automatically do so. Any tips?

<img width="424" alt="Screenshot 2025-03-17 at 1 17 34 PM" src="https://github.com/user-attachments/assets/77a00355-86b6-4247-b3df-882aa3171dd3" />